src/app/[lang]/(protected)/create-new-password/createPassForm.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import { useState, ChangeEvent, FormEvent } from "react";
+import { isValidPassword } from "@/util/input-validation";
+import { DictionaryReturnTypes } from "../../dictionaries";
+import PasswordsInput from "./passwordsInput";
+import { newPassPut } from "./newPassPut";
+import Link from "next/link";
+import Image from "next/image";
```

Comment:
Remove unused imports 
```suggestion
import { useState, FormEvent } from "react";
import { isValidPassword } from "@/util/input-validation";
import { DictionaryReturnTypes } from "../../dictionaries";
import PasswordsInput from "./passwordsInput";
import { newPassPut } from "./newPassPut";
```